### DISCUSSION TITLE
Create multiple websites with multiple domains in one repo?

### DISCUSSION THREAD

**Discussion by Moire9** [community/community#22779] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, simple question. Is it possible to have multiple different websites with multiple different domains in one GitHub repository? If I have to put them in different folders or branches, that’s okay. In fact, multiple branches would be ideal for keeping things organized. Thanks for reading.</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/sirnapkin1334">@SirNapkin1334</a>, welcome to the GitHub Support Community!</p>
<p>GitHub Pages only supports a single custom domain per repository. For your case I’d instead recommend creating separate <em>repositories</em> to host your different sites—if the content is going to be completely different across both. That way you can keep the source code for both sites separate and allow you to attach your custom domains to each independently.</p>
<p>Once you’ve uploaded your site’s content you can enable Pages from the repository settings and then add your custom domain.</p>

**Comment by IyaadArshad** [comment]
For me, the site content is the same, I see **no sense** in creating duplicate repos for the same thing, are you sure there is no alternatives?
Also: Adding redirects is not effective, my registrar is CloudFlare, and I only have 3 redirects (Page rules)
And creating another GitHub repo with a html redirect is simply going back to square one, do you have any solutions?

**Comment by Moire9** [comment]
Thank you for reminding me that this exists so I can close it as no longer necessary. I wish you luck, friend.
